modules/queue/logic/schedulerLogic.js

```diff
@@ -0,0 +1,22 @@
+const queueLogic = require('./queueLogic');
+const { MESSAGES, MESSAGE_QUEUES } = require('../constants/queue');
+
+const msInMin = 60 * 1000;
+
+const scheduledEvents = [
+  { message: MESSAGES.SCHEDULED_EVENTS.COMMANDS.CACHE_KEEPHOT, interval: process.env.KEEP_HOT_INTERVAL || 5 * msInMin, onceAtStartup: true },
+  { message: MESSAGES.SCHEDULED_EVENTS.COMMANDS.UPDATE_CHAIN_NAMES, interval: 10 * msInMin, onceAtStartup: true },
+  { message: MESSAGES.SCHEDULED_EVENTS.COMMANDS.UPDATE_TIPS_RETIPS_CLAIMS, interval: 10 * msInMin, onceAtStartup: true },
+  { message: MESSAGES.SCHEDULED_EVENTS.COMMANDS.UPDATE_EVENTS, interval: 10 * msInMin, onceAtStartup: true },
+];
+
+const SchedulerLogic = {
+  init() {
+    setTimeout(() => scheduledEvents.forEach(async ({ message, interval, onceAtStartup }) => {
```

Comment:
why wait for a second before this?

Reply:
so all listeners for these events have finished registering for sure

Reply:
did you experience any issues with this? or is it a precautionary measure?

Reply:
I think initially I had some issues, but I didn't check it again, I'll do

Reply:
It's not needed, already removed it